ZuluContent/Items/Weapons/GMWeapons/XarafaxsAxe.cs

```diff
@@ -0,0 +1,63 @@
+using Server.Engines.Harvest;
+using ZuluContent.Zulu.Engines.Magic;
+
+namespace Server.Items
+{
+    public class XarafaxsAxe : BaseAxe, IUsesRemaining, IEnchanted
+    {
+        public override HarvestSystem HarvestSystem => Lumberjacking.System;
+
+        public override int DefaultStrengthReq => 20;
+
+        public override int DefaultMinDamage => 1;
+
+        public override int DefaultMaxDamage => 15;
+
+        public override int DefaultSpeed => 40;
+
+        public override WeaponAnimation DefaultAnimation => WeaponAnimation.Bash1H;
+
+
+        [Constructible]
+        public XarafaxsAxe() : this(200)
```

Comment:
Love Xarafax's axe, I never broke one, was it only 200 uses?

Reply:
Durability was 200 on ZH scripts, need to spend more time finding out how the durability worked when harvesting on ZH, but afaik these axes never broke... it seems in RunUO code it lost a use on every mine